### ISSUE TITLE
The EMBL logo fails an accessibility audit.

### ISSUE THREAD

**Issue by sturobson** [visual-framework/vf-wp#564]
In the demo example of the `embl-logo` in the Visual Framework - https://visual-framework.github.io/vf-core//components/embl-logo/index.html

There's a line of HTML – ` <span class="vf-u-sr-only" for="text">European Molecular Biology Laboratory Home</span>`

This is required in the `vf-wp` sites - so that visitors using screen readers can tell what the `<a>` link is.

`vf-eleventy` does this already. Can this be updated please.

**Comment by dbushell** [comment]
The header/footer containers are pulled from the content hub - e.g:

https://content.embl.org/api/v1/pattern.html?filter-content-type=article&filter-id=574&pattern=node-body&source=contenthub

@khawkins98 is this something you're able to change?

On the WordPress site all we could do is edit the HTML chunck that is returned with a find/replace which is a little fragile.

**Comment by filipe-dias** [comment]
Like this?
```<header class="vf-header">
<div class="vf-global-header">
<div class="vf-global-header__inner">
<a href="/" class="embl-logo"><span class="vf-u-sr-only" for="text">European Molecular Biology Laboratory Home</span></a>```

**Comment by filipe-dias** [comment]
Ok, checked embl.org source and is like that. Changes made and the ticket can be closed
